lightkurve/targetpixelfile.py

```diff
@@ -1130,6 +1131,163 @@ def cutout(self, center=None, size=5):
+                    lc_corr = corrector_func(lc)
+                elif corrector_func == None:
+                    corrector_func = lambda x:x.normalize().remove_outliers()
+                    lc_corr = corrector_func(lc)
+                else:
+                    lc_corr = corrector_func(lc)
+
+                if periodogram == True:
+                    try:
+                        pixel_list.append(lc_corr.to_periodogram(**kwargs))
+                    except IndexError:
+                        pixel_list.append(None)
+                else:
+                    if normalize == True:
+                        if len(lc_corr.remove_nans().flux) == 0:
+                            pixel_list.append(None)
+                        else:
+                            pixel_list.append(lc_corr)
+                    elif normalize == False:
+                        if len(lc.remove_nans().flux) == 0:
+                            pixel_list.append(None)
+                        else:
+                            pixel_list.append(lc)
+
+        mask = self._parse_aperture_mask(aperture_mask)
+        tpf_plot = plt.imshow(self.flux[0])
+        plt.close()
+
+        with plt.style.context(style):
+            if title is None:
+                title = f'Target ID: {self.targetid}'
+
+            fig = plt.figure()
+            if ax == None:
+                ax = plt.gca()
+                ax.get_xaxis().set_ticks([])
+                ax.get_yaxis().set_ticks([])
+
+                if periodogram == True:
+                    ax.set(title=title, xlabel='Frequency', ylabel='Power')
+                else:
+                    if normalize == True:
+                        ax.set(title=title, xlabel='Time', ylabel='Normalized Flux')
+                    else:
+                        ax.set(title=title, xlabel='Time', ylabel='Flux')
+
+            gs = gridspec.GridSpec(self.shape[1], self.shape[2], wspace=0.01, hspace=0.01)
+
+            for k in range(self.shape[1]*self.shape[2]):
+                if pixel_list[k]:
+                    x, y = np.unravel_index(k, (self.shape[1], self.shape[2]))
+
+                    if periodogram == True:
+                        x_vals = pixel_list[k].frequency.value
+                        y_vals = pixel_list[k].power.value
+                        x_axis_min = 0
+                        x_axis_max = np.nanmax(x_vals)
+                        y_axis_min = 0
+                        y_axis_max = np.nanmax(y_vals)
+                    elif periodogram == False:
+                        x_vals = pixel_list[k].time.value
+                        y_vals = pixel_list[k].flux.value
+                        y_val_range = np.nanmax(y_vals) - np.nanmin(y_vals)
+                        x_axis_min = np.nanmin(x_vals)
+                        x_axis_max = np.nanmax(x_vals)
+                        y_axis_min = np.nanmin(y_vals) - 0.05*y_val_range
+                        y_axis_max = np.nanmax(y_vals) + 0.05*y_val_range
```

Comment:
The two lines above will indirectly trigger the following warning during the tests:
```
/home/gb/dev/lightkurve/lightkurve/targetpixelfile.py:1280: UserWarning: Attempting to set identical bottom == top == 1.0 results in singular transformations; automatically expanding.
    gax.set_ylim(y_axis_min, y_axis_max)
```

This is because the unit test uses a fake TPF (`filename_tpf_one_center`) in which the pixel values are constant over time (i.e. the pixel values in that file are always zero, except for the center pixel, which is always one).

It may not be completely unusual to have TPFs with pixels that are zero all the time, so the best workaround here may be something like this:
```suggestion
                        y_axis_min = np.nanmin(y_vals) - 0.05*max(y_val_range, 0.1)
                        y_axis_max = np.nanmax(y_vals) + 0.05*max(y_val_range, 0.1)
```